src/main/scala/mesosphere/marathon/storage/migration/MigrationTo17.scala

```diff
@@ -104,7 +104,7 @@ object MigrationTo17 extends StrictLogging {
         val reservation = rawReservation.map { raw =>
           raw.as[Reservation](InstanceMigration.legacyReservationReads(tasksMap, instanceId))
         }
-        new Instance(instanceId, Some(agentInfo), state, tasksMap, runSpecVersion, reservation)
+        new Instance(instanceId, Some(agentInfo), state, tasksMap, runSpecVersion, reservation, null)
```

Comment:
Please use `None` instead for the storage model.

Reply:
Done